src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -419,50 +399,35 @@ class OpCredsFabricTableDelegate : public chip::FabricTable::Delegate
 
-        // Remove access control entries in reverse order (it could be any order, but reverse order
-        // will cause less churn in persistent storage).
-        size_t aclCount = 0;
-        if (Access::GetAccessControl().GetEntryCount(fabricIndex, aclCount) == CHIP_NO_ERROR)
-        {
-            while (aclCount)
-            {
-                (void) Access::GetAccessControl().DeleteEntry(nullptr, fabricIndex, --aclCount);
-            }
-        }
+        NotifyFabricTableChanged();
     }
 
-    // Gets called when a fabric is loaded into the FabricTable from storage
-    void OnFabricRetrievedFromStorage(FabricTable & fabricTable, FabricIndex fabricIndex) override
-    {
-        FabricInfo * fabric = fabricTable.FindFabricWithIndex(fabricIndex);
-        // Safety check, but should not happen by the code paths involved
-        VerifyOrReturn(fabric != nullptr);
-
-        ChipLogProgress(Zcl,
-                        "OpCreds: Fabric index 0x%x was retrieved from storage. FabricId 0x" ChipLogFormatX64
-                        ", NodeId 0x" ChipLogFormatX64 ", VendorId 0x%04X",
-                        static_cast<unsigned>(fabric->GetFabricIndex()), ChipLogValueX64(fabric->GetFabricId()),
-                        ChipLogValueX64(fabric->GetNodeId()), fabric->GetVendorId());
-        fabricListChanged();
-    }
+    void OnFabricUpdated(const FabricTable & fabricTable, FabricIndex fabricIndex) override { NotifyFabricTableChanged(); }
 
     // Gets called when a fabric in FabricTable is persisted to storage
-    void OnFabricPersistedToStorage(FabricTable & fabricTable, FabricIndex fabricIndex) override
+    void OnFabricCommitted(const FabricTable & fabricTable, FabricIndex fabricIndex) override
     {
-        FabricInfo * fabric = fabricTable.FindFabricWithIndex(fabricIndex);
+        const FabricInfo * fabric = fabricTable.FindFabricWithIndex(fabricIndex);
         // Safety check, but should not happen by the code paths involved
         VerifyOrReturn(fabric != nullptr);
 
         ChipLogProgress(Zcl,
-                        "OpCreds: Fabric  index 0x%x was persisted to storage. FabricId " ChipLogFormatX64
-                        ", NodeId " ChipLogFormatX64 ", VendorId 0x%04X",
-                        static_cast<unsigned>(fabric->GetFabricIndex()), ChipLogValueX64(fabric->GetFabricId()),
-                        ChipLogValueX64(fabric->GetNodeId()), fabric->GetVendorId());
-        fabricListChanged();
+                        "OpCreds: Fabric  index 0x%x was committed to storage. Compressed Fabric Id 0x" ChipLogFormatX64
+                        ", FabricId " ChipLogFormatX64 ", NodeId " ChipLogFormatX64 ", VendorId 0x%04X",
+                        static_cast<unsigned>(fabric->GetFabricIndex()), ChipLogValueX64(fabric->GetCompressedFabricId()),
+                        ChipLogValueX64(fabric->GetFabricId()), ChipLogValueX64(fabric->GetNodeId()), fabric->GetVendorId());
+
+        NotifyFabricTableChanged();
```

Comment:
So as a followup: A commit does not actually change any attribute values, right?  The things that change attribute values are pending adds/updates, removes, and reverts.

Please don't try to address that within this PR; this has to be a followup.